spec/classes.md

```diff
@@ -1017,7 +1017,7 @@ class Test
     }
 }
 ```
-shows a nested class `M` that hides the method `M` defined in `Base`.
+shows a nested class `M` that hides the method `F` defined in `Base`.
```

Comment:
i don't believe this change is correct.  the current text is accurate.  the nested class 'M' in 'Derived' does hide the method 'M' in 'Base'.

Reply:
Agreed. This is correct as written.

Reply:
My bad, **you’re both absolutely correct**. Was wizzing through and the `class` to `method` hiding must have threw me off 🤦‍♂️ Thanks for the quick review 🙌